### PR TITLE
supports --gravity option

### PR DESCRIPTION
**Pull request by orthagonal**

### CODE REVIEW

File: lib/image.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ module.exports.compress = (file, level, callback) => {
   imagemin([file], os.tmpdir(), options).then(handleResult);
 };
 
-module.exports.crop = (im, pathToFile, origin, dims, callback) => {
+module.exports.crop = (im, pathToFile, origin, dims, gravity, callback) => {
   let gm;
   if (im) {
     gm = require('gm').subClass({ imageMagick: true });
```
Comment (jgallen23) [inline lib/image.js]: can you just always subpass and pass in true or false?

File: test/lib.test.js [hunk 2]
```diff
@@ -96,6 +96,25 @@ describe('can be used as a library', () => {
...
+      quality: 80,
+      position: [10, 10],
+      size: [120, 120],
+      gravity: 'Center'
+    };
+    s3put(stream, options, (err, response) => {
+      if (err) {
+        console.log(err);
+      }
+      chai.expect(response.key).to.include(testImageBase);
```
Comment (jgallen23) [inline test/lib.test.js]: is this actually testing anything?

File: main.js [hunk 3]
```diff
@@ -22,7 +22,7 @@ const execute = (imageFilePath, argv, callback) => {
     },
     crop: ['compress', (results, done) => {
       if (argv.size) {
-        return image.crop(argv.imagemagick, results.compress, argv.position, argv.size, (err, result) => {
+        return image.crop(argv.imagemagick, results.compress, argv.position, argv.size, argv.gravity, (err, result) => {
```
Comment (jgallen23) [inline main.js]: can we rename argv to options?  argv should just be used in CLI code

**Review by jgallen23 — CHANGES_REQUESTED** [review]

File: test/lib.test.js (hunk 2 above)
Comment (orthagonal) [reply test/lib.test.js]: Not really.  I started to write code to retrieve the image back off s3 here, to compare it to a local expected copy, but decided to just leave it.  Let me know if you want to just remove this test, or have me put in the image verifier (which we can re-use to confirm the other test cases as well).

**Pull request merged**